index.js

```diff
@@ -22,7 +22,7 @@ function load() {
 
     // The JSON data is independent of the actual file
     // hierarchy, so it is essential to extend "deeply".
```

Comment:
This comment is for the earlier code that called an `extend` function with a `deep` parameter, right? Not sure how useful it is now.